tests/__init__.py

```diff
@@ -0,0 +1,4 @@
+from tests import (  # noqa
```

Comment:
__init__.py is certainly needed- removing it gives: NotImplementedError: Can't perform this operation for unregistered loader type